docs/source/cli/model-training-inference/real-time-inference.rst

```diff
@@ -0,0 +1,320 @@
+                                 "feat_cat" : "UK"},
+                },
+                {
+                    "node_type": "author",
+                    "node_id": "a39",
+                    "features": {"feat": [-0.0032965524587780237, ......, ],
+                                "feat_cat" : "USA"},
+                },
+                ......
+            ],
+            "edges": [
+                {
+                    "edge_type": [
+                        "author",
+                        "writing",
+                        "paper"
+                    ],
+                    "src_node_id": "p4463",
+                    "dest_node_id": "p4463",
+                    "features": {"feat1": [ 1.411269339360296726, ......, ]},
+                                 "feat2" : "1980s"},
+                },
+                ......
+            ]
+        },
+        "targets": [
+            {
+                "node_type": "author",
+                "node_id": "a39"
+            }
+        ]
+    }
+
+Invoke endpoints
+****************
+There are multiple ways to invoke a Sagemaker real-time inference endpoint as documented in
+`SageMaker Developer Guide <https://docs.aws.amazon.com/sagemaker/latest/dg/realtime-endpoints-test-endpoints.html#realtime-endpoints-test-endpoints-api>`_.
+
+.. note:: GraphStorm real-time inference endpoints currently only support "application/json" as content type.
+
+Here is an example of how you can read a payload from a JSON file and use the boto3 APIs to
+invoke an endpoint.
+
+.. code-block:: python
+
+    import boto3
+    import json
+
+    # Create a SageMaker client object\n",
+    sagemaker = boto3.client('sagemaker')
+    # Create a SageMaker runtime client object using your IAM role ARN\n",
+    runtime = boto3.client('sagemaker-runtime',
+                           aws_access_key_id='your access key string',
+                           aws_secret_access_key='your secret key string',
+                           region_name='asw region' # e.g., us-east-1
+    endpoint_name='your endpoint name'    # e.g., GSF-Model4Realtime-Endpoint-2025-07-11-21-44-36
+    # load payload from a JSON file
+    with open('subg.json', 'r') as f:
+         payload = json.load(f)
+    content_type = 'application/json'
+
+    # invoke endpoint
+    response = runtime.invoke_endpoint(
+        EndpointName=endpoint_name,
+        Body=json.dumps(payload),
+        ContentType=content_type,
+        )
+    # Decodes and prints the response body
+    print(response['Body'].read().decode('utf-8'))
+
+Endpoint invoke response
+*************************
+
+As shown in the previous invoke example, the response from a GraphStorm real-time inference endpoint will include
+a JSON object in the ``Body`` field of the SageMaker API response. The details of the response syntax can be found
+in the :ref:`specification of realt-time inference response <rt-response-body-spec>`.
+
+An example of a successful inference response:
```

Comment:
Can we add a brief exlanation about the contents of "prediction" and how one could use them downstream (e.g. apply argmax)